Donation/DonationVLCConfettiView.swift

```diff
@@ -0,0 +1,77 @@
+/*****************************************************************************
+ * DonationVLCConfettiView.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <nitz.carola # googlemail.com>
+ *
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import QuartzCore
+
+public class DonationVLCConfettiView: UIView {
+
+    var emitter: CAEmitterLayer!
+    public var intensity: Float!
+    private var active: Bool!
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    func setup() {
+        intensity = 0.5
+        active = false
```

Comment:
Does `intensity` and `active` need to be an implicitly unwrapped optional?
At a first glance, it looks like you could directly set the values?